includes/classes/SiteAutomation/Request.php

```diff
@@ -176,7 +176,7 @@ private function set_status( $data ) {
 		);
 
 		if ( empty( $data['success'] ) ) {
-			$data['retry'] = $this->status['retry'] + 1;
+			$data['retry'] = is_array( $this->status ) ? $this->status['retry'] + 1 : 1;
```

Comment:
Despite the fact that checking $this->status is array will resolve the issue, but it still a potential warning if retry offset doesn't exist in the array. Could we check for `isset( $this->status['retry'] )`?

Reply:
@cadic I had similar thoughts, but checking the code I found `$this->status` is either set to a boolean or it is set to an array that always contains the `retry' key.

https://github.com/globeandmail/sophi-for-wordpress/blob/a4e6207727a7c82fc2b6d563f38f3da432bab406/includes/classes/SiteAutomation/Request.php#L169-L176

https://github.com/globeandmail/sophi-for-wordpress/blob/a4e6207727a7c82fc2b6d563f38f3da432bab406/includes/classes/SiteAutomation/Request.php#L184